### PR TITLE
i18n(es): Translate `v4.mdx`

### PR DESCRIPTION
**Pull request by Waxer59**
#### Description (required)

Translate `v4.mdx`

> [!IMPORTANT]
> There is a broken link because the section does not exist yet on that page, this section is added in #5725

### CODE REVIEW

File: src/content/docs/es/guides/upgrade-to/v4.mdx [hunk 1]
```diff
@@ -0,0 +1,387 @@
...
+  </Fragment>
+  <Fragment slot="yarn">
+  ```shell
+  # Actualiza Astro y las integraciones oficiales al mismo tiempo
+  yarn dlx @astrojs/upgrade
+  ```
+  </Fragment>
+</PackageManagerTabs>
+
+También puedes [actualizar tus integraciones de Astro manualmente](/en/guides/integrations-guide/#manual-upgrading) si es necesario, y puede que también necesites actualizar otras dependencias en tu proyecto.
```
Comment (Waxer59) [inline src/content/docs/es/guides/upgrade-to/v4.mdx:39]:
```suggestion
También puedes [actualizar tus integraciones de Astro manualmente](/en/guides/integrations-guide/#actualización-manual) si es necesario, y puede que también necesites actualizar otras dependencias en tu proyecto.
```

File: src/content/docs/es/guides/upgrade-to/v4.mdx [hunk 2]
```diff
@@ -0,0 +1,387 @@
...
+
+:::note[¿Necesitas continuar?]
+¡Después de actualizar Astro a la última versión, es posible que no necesites realizar ningún cambio en tu proyecto en absoluto!
+
+Sin embargo, si notas errores o comportamientos inesperados, por favor revisa a continuación los cambios que se han realizado y que podrían requerir actualizaciones en tu proyecto.
+:::
+
+Astro v4.0 incluye [cambios potencialmente importantes](#breaking-changes), así como la [eliminación de algunas funciones obsoletas](#características-previamente-obsoletas-ahora-eliminadas).
```
Comment (Waxer59) [inline src/content/docs/es/guides/upgrade-to/v4.mdx:47]:
```suggestion
Astro v4.0 incluye [cambios potencialmente importantes](#cambios-importantes), así como la [eliminación de algunas funciones obsoletas](#características-previamente-obsoletas-ahora-eliminadas).
```

File: src/content/docs/es/guides/upgrade-to/v4.mdx [hunk 3]
```diff
@@ -0,0 +1,387 @@
...
+  </Fragment>
+  <Fragment slot="yarn">
+  ```shell
+  # Actualiza Astro y las integraciones oficiales al mismo tiempo
+  yarn dlx @astrojs/upgrade
+  ```
+  </Fragment>
+</PackageManagerTabs>
+
+También puedes [actualizar tus integraciones de Astro manualmente](/en/guides/integrations-guide/#actualización-manual) si es necesario, y puede que también necesites actualizar otras dependencias en tu proyecto.
```
Comment (dreyfus92) [inline src/content/docs/es/guides/upgrade-to/v4.mdx:39]:
```suggestion
También puedes [actualizar tus integraciones de Astro manualmente](/es/guides/integrations-guide/#actualización-manual) si es necesario, y puede que también necesites actualizar otras dependencias en tu proyecto.
```

**Review by dreyfus92 — CHANGES_REQUESTED** [review]
Buen trabajo @Waxer59, te dejo una sugerencia 😁

**Comment by Waxer59** [comment]
Gracias por la revisión 🤠

**Review by sarah11918 — APPROVED** [review]
Thank you, @waxer59 !

**Pull request merged**